hedera-node/hedera-app/src/main/java/com/hedera/node/app/store/ReadableStoreFactory.java

```diff
@@ -114,6 +117,7 @@ private static Map<Class<?>, StoreEntry> createFactoryMap() {
         newMap.put(
                 ReadablePlatformStateStore.class,
                 new StoreEntry(PlatformStateService.NAME, ReadablePlatformStateStore::new));
+        newMap.put(RosterStateAccessor.class, new StoreEntry(RosterStateId.NAME, ReadableRosterStore::new));
```

Comment:
note to reviewers: It's not clear to me whether or not to provide this as the instance of the accessor required by the platform to start has already been created outside of this factory. However, all stores, regardless of whether they're used before or after this factory gets initialized, are defined here so I have added a definition here too. cc @tinker-michaelj.